api/proto/node/node.proto

```diff
@@ -33,6 +33,23 @@ service Retrieval {
 	rpc GetBlobHeader(GetBlobHeaderRequest) returns (GetBlobHeaderReply) {}
 	// Retrieve node info metadata
 	rpc NodeInfo(NodeInfoRequest) returns (NodeInfoReply) {}
+
+	////////////////////////////////////////////////////////////////////////////////////////////////
+	// These RPCs are a work in progress and are not yet fully functional. API may not be stable. //
+	////////////////////////////////////////////////////////////////////////////////////////////////
+
+	// GetChunks retrieves all of the chunks from a specified quorum for a blob held by the node.
+	// Eventually this will replace RetrieveChunks.
+	rpc GetChunks(GetChunksRequest) returns (GetChunksReply) {}
+	// GetChunk retrieves a specific chunk for a blob custodied at the Node.
+	rpc GetChunk(common.ChunkKey) returns (common.ChunkData) {}
+	// GetHeader is similar to RetrieveChunks, this just returns the header of the blob.
+	// Eventually this will replace GetBlobHeader.
+	rpc GetHeader(common.BlobKey) returns (GetBlobHeaderReply) {}
+	// StreamHeaders requests a stream all new headers.
+	rpc StreamHeaders(stream StreamHeadersRequest) returns (stream GetBlobHeaderReply) {}
+	// Retrieve node info metadata. Eventually this will replace NodeInfo.
+	rpc GetNodeInfo(GetNodeInfoRequest) returns (GetNodeInfoReply) {}
```

Comment:
Why do we need a new node info API?

Reply:
The main goal is to have better uniformity in the names of the RPCs. This does the same thing as the RPC `NodeInfo`, it just has a different name. Should we remove this new RPC for now?